react/src/machine-coding/traffic-light/App.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import styles from './trafficlight.module.css';
+import config from './config';
+const App = () => {
+  const [currentActiveLight, setCurrentActiveLight] = useState('red');
+  useEffect(() => {
+    let timerId;
+    const { duration, next } = config[currentActiveLight];
+    timerId = setTimeout(() => {
+      setCurrentActiveLight(next);
+    }, duration);
+    return () => clearTimeout(timerId);
+  }, [currentActiveLight]);
+  return (
+    <div className={styles.AppContainer}>
+      <div className={styles.trafficWrapper}>
+        {Object.keys(config).map((light, index) => {
+          return (
+            <div
+              className={styles.light}
+              style={{ background: `${currentActiveLight === light ? `${light}` : ''}` }}
+            ></div>
+          );
+        })}
```

Comment:
Each item in list-rendering should have a unique `key` prop. Please add the `key` prop in the list elements.

Note that the `index` variable is unused. Please remove it if it is not required. Also, ideally, index should not be used as key.